columnq/Cargo.toml

```diff
@@ -36,10 +36,10 @@ calamine = "0.19.1"
 tokio = { version = "1", features = ["rt-multi-thread"] }
 futures = "0.3"
 # S3
-rusoto_core = { version = "0.47", default-features = false }
-rusoto_s3 = { version = "0.47", default-features = false }
-rusoto_credential = { version = "0.47" }
-rusoto_sts = { version = "0.47", default-features = false }
+rusoto_core = { version = "0.48", default-features = false }
```

Comment:
need upgrade, since `Cargo.lock` contains both versions. 

ELSE two versions point to different `hyper-rustls` (0.23.0 vs 0.22.1), Rust compile complains about the type

```
note: struct `HttpsConnector` and struct `hyper_rustls::connector::HttpsConnector` have similar names, but are actually distinct types
note: struct `HttpsConnector` is defined in crate `hyper_rustls`
   --> /Users/******/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-rustls-0.23.0/src/connector.rs:20:1
    |
20  | pub struct HttpsConnector<T> {
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
note: struct `hyper_rustls::connector::HttpsConnector` is defined in crate `hyper_rustls`
   --> /Users/******/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-rustls-0.22.1/src/connector.rs:21:1
```